src/Servers/IIS/src/AspNetCoreModuleV2/AspNetCore/dllmain.cpp

```diff
@@ -102,31 +102,7 @@ HRESULT
         g_hEventLog = RegisterEventSource(nullptr, ASPNETCORE_EVENT_PROVIDER);
     }
 
-    // check whether the feature is disabled due to security reason
-    if (RegOpenKeyEx(HKEY_LOCAL_MACHINE,
-        L"SOFTWARE\\Microsoft\\IIS Extensions\\IIS AspNetCore Module V2\\Parameters",
-        0,
-        KEY_READ,
-        &hKey) == NO_ERROR)
-    {
-        DWORD dwType = 0;
-        DWORD dwData = 0;
-        DWORD cbData;
-
-        cbData = sizeof(dwData);
-        if ((RegQueryValueEx(hKey,
-            L"DisableANCM",
-            nullptr,
-            &dwType,
-            (LPBYTE)&dwData,
-            &cbData) == NO_ERROR) &&
-            (dwType == REG_DWORD))
-        {
-            fDisableANCM = (dwData != 0);
-        }
-
-        RegCloseKey(hKey);
-    }
+    fDisableANCM = RegistryKey::TryGetDWORD(HKEY_LOCAL_MACHINE, L"SOFTWARE\\Microsoft\\IIS Extensions\\IIS AspNetCore Module V2\\Parameters", L"DisableANCM").value_or(0) != 0;
```

Comment:
Confused about how this compiles; don't you need a 4th parameter for DWORD flags here?

Reply:
Default value in header file https://github.com/aspnet/AspNetCore/pull/5705/files#diff-483217b71a7366c31a9bd9e93f05d981R13